### ISSUE TITLE
`create_constant` is broken

### ISSUE THREAD

**Issue by lockshaw** [flexflow/FlexFlow#349]
@soumyac1999 Can you share a command to reproduce the issue you ran into and a copy of the error?

**Comment by soumyac1999** [comment]
Code to reproduce (create_constant_test.py):
```python
import flexflow.core as ff
from flexflow.type import DataType

def top_level_task():
    ffconfig = ff.FFConfig()
    ffmodel = ff.FFModel(ffconfig)

    d = ffmodel.create_constant([1], 1, DataType.DT_INT32)

if __name__ == "__main__":
  top_level_task()
```

Command to run: `flexflow_python create_constant_test.py -ll:py 1 -ll:gpu 1 -ll:fsize 2048 -ll:zsize 12192`

I think I am passing incorrect parameters to `create_constant` but https://github.com/flexflow/FlexFlow/blob/master/src/runtime/model.cc#L1167 will fail in any case.

Log:
```
[0 - 7f0af8646000]    0.244210 {3}{Mapper}: Enabled Control Replication Optimizations.
[0 - 7f0af8646000]    0.244348 {3}{Mapper}: Enabled Control Replication Optimizations.
[0 - 7f0af8646000]    0.244394 {3}{Mapper}: Enabled Control Replication Optimizations.
Using cffi flexflow bindings.
Using flexflow python
start top-level task
top-level task
Using flexflow backend.
workSpaceSize (1024 MB)
flexflow_python: /home/soumya/flexflow/src/runtime/model.cc:1167: FlexFlow::TensorBase* FlexFlow::FFModel::create_constant(const int*, float, DataType) [with int NDIM = 1; FlexFlow::Tensor = FlexFlow::TensorBase*]: Assertion `false' failed.
Aborted
```

**Comment by jiazhihao** [comment]
The assertion failure should have been fixed by #390
